Update_CRO_integreatly-operator.md

```diff
@@ -1,4 +1,4 @@
-# Adding Latest CRO to RHMI
+# Adding Latest CRO to RHOAM
```

Comment:
@cecobask , actually same question, as in README, - name of  package: "rhmi-cloud-resources" in Integrately.